### PR TITLE
User Dashboard & User Card components created

### PR DESCRIPTION
**Pull request by rheactdev**
# What's here (& why)

User Dashboard to view all current users in the system initially created: https://trello.com/c/wDmJVykp/25-as-an-admin-i-can-see-all-current-users-in-the-system-in-order-to-manage-them

# Type of Change

- [ ]  Bug fix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [x] Tested locally
- [x] Test suite provides the same results as prior to making changes

# Checklist

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] My changes generate no new warnings
- [ x] New and existing unit tests pass locally with my changes 
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/UserDashboard/UserCard.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// id, firstname, lastname, role, email
+export default function UserCard({ user }) {
+  // console.log(user)
```
Comment (chazkiker2) [inline src/components/UserDashboard/UserCard.js]: Let's delete commented-out code.

File: src/components/UserDashboard/UserCard.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// id, firstname, lastname, role, email
```
Comment (chazkiker2) [inline src/components/UserDashboard/UserCard.js]: Let's delete commented-out code.

File: src/state/ducks/userDuck.js [hunk 3]
```diff
@@ -53,6 +54,25 @@ export const userActions = {
...
+      thunkFail,
+      thunkResolve,
+    } = userThunkUtils.getTriggersFromPrefix(dispatch, 'get-all-users');
+
+    thunkStart();
+
+    axiosAuth()
+      .get('/users/users')
+      .then(res => {
+        // console.log(res.data);
```
Comment (chazkiker2) [inline src/state/ducks/userDuck.js]: delete commented-out code.

File: src/state/ducks/userDuck.js [hunk 4]
```diff
@@ -103,6 +123,16 @@ const userReducer = (state = initialState, action) => {
           role: roles[0].role.name,
         },
       };
+
+    case GET_ALL_USERS_SUCCESS:
+      // console.log(action.payload)
```
Comment (chazkiker2) [inline src/state/ducks/userDuck.js]: delete commented-out code.

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
Code looks great! 

Only request is that we delete all commented-out code to keep that codebase clean. 

Will approve promptly after those are deleted :).


Nice work!

**Review by chazkiker2 — APPROVED** [review]
You killed it!! (And what a perfect `thunk`!!!)

**Review by sebass482 — APPROVED** [review]
looks good, comments removed. good stuff Rhea

**Pull request merged**